core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -2,7 +2,7 @@
 <ng-template #enabledComponent>
     <dotcms-template-builder-lib
         [layout]="pageState.layout"
-        [themeId]="pageState.template.theme"
+        [template]="pageState.template"
```

Comment:
What I meant was that instead of adding a new `@Input()` to the `dotcms-template-builder-lib` we can just pass a new property (theme id) in the `layout`.